src/mkdocstrings_handlers/asp/handler.py

```diff
@@ -56,12 +61,14 @@ def collect(self, identifier: str, config: dict) -> dict | None:
         document = Document.new(path, content)
 
         data = {
+            "project_name": project_data["project"]["name"],
```

Comment:
Added this to have links in the encodings, we might want to not complicate things in the future, but i think it is nice